docs/core/deploying/trimming/fixing-warnings.md

```diff
@@ -84,24 +84,29 @@ foreach (var m in type.GetMethods())
 }
 ```
 
-In the example above, the real problem is `Console.ReadLine()`. Because *any* type could be read, the trimmer has no way to know if you need methods on `System.Tuple` or `System.Guid` or any other type. On the other hand, the following code would be fine:
+In the example above, the real problem is `Console.ReadLine()`. Because *any* type could be read, the trimmer has no way to know if you need methods on `System.String` or `System.Guid` or any other type. On the other hand, the following code would be fine:
 
 ```csharp
-Type type = typeof(System.Tuple);
+Type type = typeof(string);
 foreach (var m in type.GetMethods())
 {
     Console.WriteLine(m.Name);
 }
 ```
 
-Here the trimmer can see the exact type being referenced: `System.Tuple`. Now it can use flow analysis to determine that it needs to keep all public methods on `System.Tuple`. So where does `DynamicallyAccessMembers` come in? When reflection is split across multiple methods.
+Here the trimmer can see the exact type being referenced: `System.String`. Now it can use flow analysis to determine that it needs to keep all public methods on `System.String`. So where does `DynamicallyAccessMembers` come in? When reflection is split across multiple methods. In the below code, we can see that the type `string` flows to `Method3` where reflection is used to access `string`'s methods,
```

Comment:
Maybe I'm just paranoid - I fear that the usage of `string` could be confusing. Mainly because we allow DAM annotations on `string param` as well. Can we we choose some other type for this?
(also `string` is rather special in various ways... )

Reply:
That's a good point, I've updated to use System.DateTime instead.